src/components/Intranet/Users/UserInfoModal.tsx

```
@@ -60,37 +63,66 @@ export default function UserInfoModal({
         }
     }, [userId, fetchUser]);
 
+    useEffect(() => {
+        setImageLoaded(false);
+        setImageError(false);
+    }, [userId]);
+
+    useEffect(() => {
+        if (user && user.image_url && !imageLoaded && !imageError) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)